python/tskit/trees.py

```diff
@@ -7956,9 +7970,9 @@ def allele_frequency_spectrum(
             window (defaults to True).
         :return: A (k + 1) dimensional numpy array, where k is the number of sample
             sets specified.
+            If there is one sample set and windows=None, a 1 dimensional array is
+            returned.
```

Comment:
Why do we need this note?

Reply:
Because this
```
A (k + 1) dimensional numpy array, where k is the number of sample
            sets specified
```
is not true in this case, right? If it was it would be a 2D array.

Reply:
oh hah right, thanks. =)